--- fastnlotoolkit/include/fastnlotk/fastNLOCoeffAddBase.h.orig	2022-04-06 16:06:44.993363517 +0200
+++ fastnlotoolkit/include/fastnlotk/fastNLOCoeffAddBase.h	2022-04-06 16:08:27.560330160 +0200
@@ -115,17 +115,17 @@
    // Manipulate coefficient bins
    virtual void Clear();//!< Clear all coefficients and event counters
    virtual void NormalizeCoefficients(double wgt=1); //!< Set number of events to unity and normalize coefficients accordingly
-   virtual void NormalizeCoefficients(const std::vector<std::vector<double> >& wgtProcBin) {};
-   virtual void MultiplyCoefficientsByConstant(double fact) {};//!< Multiply all coefficients of all bins by a constant factor
-   virtual void MultiplyBin(unsigned int iObsIdx, double fact) {};  //!< Multiply coefficients of one observable bin a factor
-   virtual void MultiplyBinProc(unsigned int iObsIdx, unsigned int iProc, double fact) {}; //!< Multiply coefficients of one observable bin a factor (idx starting from 0)
+   virtual void NormalizeCoefficients(const std::vector<std::vector<double> >& /*wgtProcBin*/) {};
+   virtual void MultiplyCoefficientsByConstant(double /*fact*/) {};//!< Multiply all coefficients of all bins by a constant factor
+   virtual void MultiplyBin(unsigned int /*iObsIdx*/, double /*fact*/) {};  //!< Multiply coefficients of one observable bin a factor
+   virtual void MultiplyBinProc(unsigned int /*iObsIdx*/, unsigned int /*iProc*/, double /*fact*/) {}; //!< Multiply coefficients of one observable bin a factor (idx starting from 0)
    virtual void EraseBin(unsigned int iObsIdx, int ITabVersionRead);//!< Erase observable bin from table
    virtual void CatBin(const fastNLOCoeffAddBase& other, unsigned int iObsIdx, int ITabVersionRead); //!< Catenate observable to table
 
    int GetIRef() const {return IRef;}
    void SetIRef(int iref=1) {IRef=iref;}
    double GetNevt() const { return Nevt; }
-   double GetNevt(int NObsBin, int NSubproc) const {
+   double GetNevt(int /*NObsBin*/, int /*NSubproc*/) const {
       //return fWgt.WgtObsSumW2[NSubproc][NObsBin];
       if (Nevt > 0) return Nevt;
       else {std::cout<<"Todo. Preparation for v2.3."<< std::endl; return Nevt;}
